.gitignore

```diff
@@ -25,3 +25,4 @@ go.work
 *.swp
 *.swo
 *~
+scripts/__pycache__/*
```

Comment:
Let's move all the scripts inside the `hack/` folder, as it's a common pattern in golang projects it would be a less surprising place to find them